util/cron/nightly

```diff
@@ -411,7 +411,7 @@ if ($mailer eq "") {
 print "\$mailer = $mailer\n";
 
 
-$launchcmd = "$ENV{'CHPL_APP_LAUNCH_CMD'}";
+$launchcmd = "$ENV{'CHPL_TEST_LAUNCHCMD'}";
```

Comment:
Nice, I have never understood the `CHPL_APP_LAUNCH_CMD` bit. That said, start_test will read `CHPL_TEST_LAUNCHCMD` so I'm not sure nightly needs to read it and then throw `--lanchcmd` to start_test too

Reply:
Ah, that's a good point. Looking a bit to it, it seems like nightly is passing that argument directly all the way to start_test without anything touching it in between (paratest.*).

As usual, I am scared to grow the scope of the PR, though :) Do you have any problem if that comes soon after we get everything else right?

Reply:
That's fine. I think we just end up specify launchcmd in 2 different ways (start_test flag and env var that start_test reads), but that shouldn't cause any problems.

Reply:
Yeah. That's what I am saying. We'll be basically overriding the default of that flag... with the default of it